common/math/math.go

```diff
@@ -36,16 +40,35 @@ func CalculateFee(amount, fee float64) float64 {
 	return amount * (fee / 100)
 }
 
-// CalculatePercentageGainOrLoss returns the percentage rise over a certain
-// period
-func CalculatePercentageGainOrLoss(priceNow, priceThen float64) float64 {
-	return (priceNow - priceThen) / priceThen * 100
+// PercentageChange returns the percentage change between two numbers, v1 is reference value.
+func PercentageChange(v1, v2 float64) float64 {
```

Comment:
nitpick: Consider `x`, `y` instead of `v1`, `v2`.
[Rationale](https://pkg.go.dev/math)

Reply:
[whoop](https://github.com/thrasher-corp/gocryptotrader/pull/1617/commits/bdc96463b2f21055eb9765faa400d7d72bc331ac)